tensorboard/notebook.py

```diff
@@ -378,18 +379,35 @@ def _display_ipython(port, height, display_handle):
       <script>
         (function() {
           const frame = document.getElementById(%JSON_ID%);
-          const url = new URL("/", window.location);
-          url.port = %PORT%;
+          const url = new URL(%URL%, window.location);
+          const port = %PORT%;
+          if (port) {
```

Comment:
OK—this branches on `0` as well, but `0` isn’t a valid port for opening
a connection, so this is fine with me. No action required.